inscriptions/ISic001157.xml

```diff
@@ -190,22 +195,21 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140642">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Θεοῖς Κατ<supplied reason="lost">αχ</supplied>-
-                    <lb n="2"/>
-                    <supplied reason="omitted">θ</supplied>ονίοις.<lb n="3"/>ΜΗΚΑΛΙ Σουλ-<lb n="4"/>πίκιος Ἀνί-<lb n="5"/>κητος ἔζη-<lb n="6"/>σεν ἔτη XXXX.
-                    <lb n="7"/>
+                    <lb n="1"/>θεοῖς Κατ<supplied reason="lost">αχ</supplied>
+                    <lb n="2"/><corr><sic>ε</sic><reg>θ</reg></corr>ονίοις.
```

Comment:
again, I'd just put the E vs θ in the apparatus, with the θ in the main text
don't forget to add 'break="no"' on the lb at the start of line 2